app/models/provider.rb

```diff
@@ -99,7 +99,7 @@ def new_draft_attributes
            primary_key: :provider_code,
            inverse_of: :accrediting_provider
 
-
+  has_many :accrediting_providers, through: :courses
```

Comment:
`:accredited_bodies`?

Reply:
I'd have to fix all the naming to do that

Reply:
Card it up perhaps? It would also affect frontend as it would change the api surface

Reply:
Is this ++good because it differentiates it better from the `Provider` model?